dev/ci/cases/yamls/gfs_defaults_ci.yaml

```
@@ -1,5 +1,5 @@
 defaults:
-  !INC {{ HOMEgfs }}/parm/config/gfs/yaml/defaults.yaml
+  !INC {{ HOMEgfs }}/dev/dev/parm/config/gfs/yaml/defaults.yaml
```

Extra `/dev` snuck in.
```suggestion
  !INC {{ HOMEgfs }}/dev/parm/config/gfs/yaml/defaults.yaml
```

---

Thanks @KateFriedman-NOAA. Fixed.